jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/server/impl/DashbuilderBootstrap.java

```diff
@@ -131,16 +136,17 @@ protected void registerDataSetDefinitions() {
                 .label(DataSetTasksListGridViewImpl.COLUMN_TASKID)   //declaring as label(even though it's numeric) because needs apply groupby and  Group by number not supported
                 .number(DataSetTasksListGridViewImpl.COLUMN_WORKITEMID)
                 .label(DataSetTasksListGridViewImpl.COLUMN_ORGANIZATIONAL_ENTITY)
+                .label( DataSetTasksListGridViewImpl.COLUMN_ISFORADMIN )
                 .buildDef();
 
         DataSetDef humanTaskWithAdminDef = DataSetFactory.newSQLDataSetDef()
                 .uuid(HUMAN_TASKS_WITH_ADMIN_DATASET)
                 .name("Human tasks and admins")
                 .dataSource(JBPM_DATASOURCE)
-                .dbSQL("select t.activationTime, t.actualOwner, t.createdBy, "
-                        + "t.createdOn, t.deploymentId, t.description, t.dueDate, "
-                        + "t.name, t.parentId, t.priority, t.processId, t.processInstanceId, "
-                        + "t.processSessionId, t.status, t.taskId, t.workItemId, oe.id oeid "
+                .dbSQL("select t.activationtime, t.actualowner, t.createdby, "
```

Comment:
I'll leave the complete review to the authors of this code (@nmirasch). But this is another place where you're rewriting the query to lowercase column names from CamelCase. Could you change it here as well, please?